js/projects_info/projects_info.js

```diff
@@ -4,23 +4,16 @@ var projects = [];
 To add a new project, add a new item to blogsInfo and push that item to projects.
 */
 
-var projectsInfo = {
-  'testProject': {
-    'name': "Gav\'s Dev Blog",
-    'imageLocation': './images/blog_images/gavs_dev_blog.jpg',
-    'link': 'https://gavsdevblog.wordpress.com/',
-    'description': "New Description",
-    'languages': ["HTML", "CSS", "JS"]
-  },
-  'scoutApp': {
+var projectsInfo = [
+  {
     'name': "Scout-App",
     'imageLocation': './images/project_images/scout-app.jpg',
     'link': 'http://scout-app.io/',
     'description': "Scout-App, the easiest way to use Sass!",
-    'languages': ["jQuery", "JS", "Node", "sass", "HTML"]
+    'languages': ["jquery", "javascript", "nodejs", "sass", "html5"]
```

Comment:
I think "languages" is very limiting and not very accurate. "technologies" would probably be a better name. Also `NW.js` should be added to the list.